.npmignore

```diff
@@ -0,0 +1,4 @@
+.tsconfig.json
+test/
+src/
+.gitignore
```

Comment:
This file should be deleted. The `files` property in `package.json` will do its job, and in a more robust manner. I recommend testing which files are published with `npm pack --dry-run`.

(`pack` and `publish` are the two remaining `npm` commands in my workflow.)